setup/requirements.md

```diff
@@ -49,12 +49,12 @@ The use of Coder deployments requires a license that's emailed to you.
 
 ### Restrictions
 
-Deployments using the free level of Coder (1-10 users):
+Deployments using the free 60-day free trial of Coder:
```

Comment:
`free 60-day free trial of Coder`

free is mentioned twice in an awkward manner